security-advisories/mbedtls-security-advisory-2024-08-1.md

```diff
@@ -0,0 +1,24 @@
+# CTR\_DRBG prioritized over HMAC\_DRBG as the PSA DRBG
+
+**Title** | CTR\_DRBG prioritized over HMAC\_DRBG as the PSA DRBG
+--------- | -----------------------------------------------------
+**CVE** | TODO
+**Date** | TODO
```

Comment:
We might be still waiting for the CVE, but I think we know the date now. (Same for the other two.)

Reply:
The CVE IDs are in the changelog.

Reply:
Apologies, I forgot that there were updates to do. I have filled in both CVE and date.